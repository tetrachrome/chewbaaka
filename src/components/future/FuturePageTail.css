/**
 * FuturePageTail.css
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 26, 2020
 * Updated  : Aug 11, 2020
 */

div.FuturePageTailOuterContainer {
  margin: 0 auto;
  width: 100%;
}

div.FuturePageTailStatementBoxOuterContainer {
  border: 24px solid teal;  
}

div.FuturePageTailStatementBoxInnerContainer {
  background-color: skyblue;
}

.FuturePageTailTitle {
  position: relative;
  top: 60px;
  margin: 60px auto;
  font-family: Arial, Helvetica, sans-serif;
  font-size: 32px;
  font-weight: 900;
}

div.FuturePageTailVisionStatementTextContainer {
  display: flex;
}

p.FuturePageTailVisionStatementText {
  width: 60%;
  max-width: 600px;
  margin: auto;
  margin-top: 60px;
  margin-bottom: 60px;
  text-align: center;
  font-family: cursive;
  font-size: 28px;
}
