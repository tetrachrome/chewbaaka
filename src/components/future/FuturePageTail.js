/**
 * FuturePageTail.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 26, 2020
 * Updated  : Jul 26, 2020
 */

import React from 'react'

import ContentPageSubsectionTemplate from '../shared/ContentPageSubsectionTemplate'

import {
  kStringConstantCheetahConservationFund,
  kStringConstantCCFVisionStatement
} from '../shared/constants'

import './FuturePageTail.css'

export default class FuturePageTail extends React.Component {

  render() {
    return (
      <ContentPageSubsectionTemplate
        content={this.renderBody()}
      />
    );
  }

  renderBody() {
    return (
      <div className="FuturePageTailOuterContainer">
        <div className="FuturePageTailStatementBoxOuterContainer">
          <div className="FuturePageTailStatementBoxInnerContainer">
            <h5 className="FuturePageTailTitle TextCentered">
              {`${kStringConstantCheetahConservationFund} Vision Statement`}
            </h5>
            <p className="FuturePageTailVisionStatementText TextCentered">
              {kStringConstantCCFVisionStatement}
            </p>
          </div>
        </div>
      </div>
    );
  }
}
