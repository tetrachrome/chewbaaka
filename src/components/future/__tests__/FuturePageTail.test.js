/**
 * FuturePageTail.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 26, 2020
 * Updated  : Jul 26, 2020
 */

import React from 'react'

import { render } from '@testing-library/react'

import renderer from 'react-test-renderer'

import FuturePageTail from '../FuturePageTail'

test('renders FuturePageTail component', () => {
  render(
    <FuturePageTail />
  );
});

test('FuturePageTail component snapshot', () => {
  const tree = renderer
    .create(
      <FuturePageTail />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
