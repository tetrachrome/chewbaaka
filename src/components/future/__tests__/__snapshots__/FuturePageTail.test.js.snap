// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FuturePageTail component snapshot 1`] = `
<div
  className="ContentPageSubsectionTemplateOuterContainer ContentPageSubsectionTemplateOuterContainer_DebugColor DebugWireframe"
>
  <div
    className="ContentPageSubsectionTemplateInnerContainer ContentPageSubsectionTemplateInnerContainer_DebugColor DebugWireframe ContentPageContentInnerContentDimension ContentPageContentInnerContentDimension_DebugColor DebugWireframe"
  >
    <h3
      className="ContentPageSubsectionTitle ContentPageSubsectionTitle_DebugColor DebugWireframe"
    />
    <div
      className="FuturePageTailOuterContainer"
    >
      <div
        className="FuturePageTailStatementBoxOuterContainer"
      >
        <div
          className="FuturePageTailStatementBoxInnerContainer"
        >
          <h5
            className="FuturePageTailTitle TextCentered"
          >
            Cheetah Conservation Fund Vision Statement
          </h5>
          <p
            className="FuturePageTailVisionStatementText TextCentered"
          >
            We see a world in which cheetahs live and flourish in coexistence with people and the environment.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
